test/gerber-parser_drill_test.js

```diff
@@ -379,6 +379,28 @@ describe('gerber parser with gerber files', function() {
 
       p.write('X1Y1\n')
     })
+
+    it('should warn / detect trailing zeros if possible', function(done) {
+      var warnings = []
+
+      p.on('warning', function(w) {
+        warnings.push(w)
+      })
+
+      p.on('finish', function() {
+        expect(warnings[0].message).to.match(/assuming trailing/)
+        expect(warnings[1].message).to.match(/detected leading/)
+        expect(p.format.zero).to.equal('L')
+        done()
+      })
+
+      p.write(';FILE_FORMAT=2:4\n')
+      p.write('INCH\n')
+      p.write('G90\n')
+      p.write('T1C0.015\n')
+      p.write('X0.7550Y1.4000\n')
```

Comment:
Sorry, I don't really know what I am doing. So what I wrote actually has a leading zero but the decimal dots are not needed (but don't hurt)?